backend/tracim_backend/views/contents_api/file_controller.py

```diff
@@ -55,11 +62,67 @@ class FileController(Controller):
     """
 
     # File data
+    @hapic.with_api_doc(tags=[SWAGGER_TAG__FILE_ENDPOINTS])
+    @require_workspace_role(UserRoleInWorkspace.CONTRIBUTOR)
+    @hapic.input_path(WorkspaceIdPathSchema())
+    @hapic.output_body(ContentDigestSchema())
+    @hapic.input_forms(FileCreationFormSchema())
+    @hapic.input_files(SimpleFileSchema())
+    def create_file(self, context, request: TracimRequest, hapic_data=None):
+        """
+        Create a file .This will create 2 new
+        revision.
+        """
+        app_config = request.registry.settings['CFG']
+        api = ContentApi(
+            show_archived=True,
+            show_deleted=True,
+            current_user=request.current_user,
+            session=request.dbsession,
+            config=app_config,
+        )
+        file = hapic_data.files.files
```

Comment:
do not name a local var `file`. Good habit is to name it f or _file... by the way, why do you name it file while in hapic_data its files (plural form)?